project/proposals/119-custom-connectors.md

```diff
@@ -42,12 +42,13 @@ properties.
 
 New API endpoints for defining custom connectors:
 
-| HTTP Verb | Path                                            | Description                                                                     |
-| --------- | ----------------------------------------------- | ------------------------------------------------------------------------------- |
-| GET       | /api/**{version}**/connector-templates          | Returns a list of known connector templates                                     |
-| GET       | /api/**{version}**/connector-templates/**{id}** | Returns a specific connector template identified by the given **id**            |
-| POST      | /api/**{version}**/connectors/custom/info       | Provides information like proposed name, icon and description for new connector |
-| POST      | /api/**{version}**/connectors/custom/           | Create a new custom connector                                                   |
+| HTTP Verb | Path                                                             | Description                                                                           |
+| --------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| GET       | /api/**{version}**/connector-templates                           | Returns a list of known connector templates                                           |
+| GET       | /api/**{version}**/connector-templates/**{templateId}**          | Returns a specific connector template identified by the given **templateId**          |
+| POST      | /api/**{version}**/connectors/custom/info                        | Provides information like proposed name, icon and description for new connector       | 
+| POST      | /api/**{version}**/connectors/custom/                            | Create a new custom connector                                                         |
+| GET       | /api/**{version}**/connectors/custom?templateId=**{templateId}** | Lists all connectors that were created from a template identified with **templateId** |
```

Comment:
I think we should do this under `/connectors` and just leave the `/connectors/custom` as a subresource convenience to create custom connectors from templates.